LEAF_Request_Portal/admin/templates/mod_workflow.tpl

```diff
@@ -6,6 +6,7 @@
     <div id="btn_newWorkflow" class="buttonNorm" onclick="newWorkflow();" style="font-size: 120%" role="button" tabindex="0"><img src="../../libs/dynicons/?img=list-add.svg&w=32" alt="New Workflow" /> New Workflow</div><br />
     <br />
     <div id="btn_deleteWorkflow" class="buttonNorm" onclick="deleteWorkflow();" style="font-size: 100%; display: none" role="button" tabindex="0"><img src="../../libs/dynicons/?img=list-remove.svg&w=16" alt="Delete workflow" /> Delete workflow</div><br />
+    <div id="btn_listActionType" class="buttonNorm" onclick="listActionType();" style="font-size: 100%; display: none" role="button" tabindex="0">List Action Type</div><br />
```

Comment:
Should probably change "List Action Type" to "Edit Actions".

In LEAF, there is a difference between "action type" and "action", so "Type" should be left out. This also applies to any methods/variables used when editing Actions.

Also, the default actions that ship with LEAF should not be editable/deletable, I think there are seven of them.